### DISCUSSION TITLE
Shopping List Organization

### DISCUSSION THREAD

**Discussion by BonzTM** [mealie-recipes/mealie#1634] [Feature Request]
**Before submitting this feature request, I have**

- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe the Problem to Be Solved**

The shopping list feature is unable to be re-organized.  My preference would be for it to be easy via mobile (drag and drop).  I'd like to replace my Trello shopping lists with this functionality, but my wife likes to drag-and-drop organize the list based on the route we shop through the store.

**(Optional): Suggest A Solution**

**Additional Information**

**Comment by michael-genson** [comment]
This has since been implemented
